google/cloud/bigquery_storage_v1beta2/client.py

```diff
@@ -135,3 +141,58 @@ def read_rows(
             offset,
             {"retry": retry, "timeout": timeout, "metadata": metadata},
         )
+
+
+class BigQueryWriteClient(big_query_write.BigQueryWriteClient):
+    """BigQuery Write API.
+
+    The Write API can be used to write data to BigQuery.
+    """
+
+    def append_rows(
+        self,
+        initial_request: types.AppendRowsRequest,
+        # TODO: add retry argument. Blocked by
```

Comment:
Good point. I've added a `*` to force `timeout` and `metadata` to be keyword arguments only. This is the approach the generated clients have been taking to avoid argument order being a part of the contract.